src/Simplex/Chat/Store/Messages.hs

```diff
@@ -548,16 +550,11 @@ data ChatPreviewData (c :: ChatType) where
 
 data AChatPreviewData = forall c. ChatTypeI c => ACPD (SChatType c) (ChatPreviewData c)
 
-paginationByTimeFilter :: PaginationByTime -> (Query, [NamedParam])
-paginationByTimeFilter = \case
-  PTLast count -> ("\nORDER BY ts DESC LIMIT :count", [":count" := count])
-  PTAfter ts count -> ("\nAND ts > :ts ORDER BY ts ASC LIMIT :count", [":ts" := ts, ":count" := count])
-  PTBefore ts count -> ("\nAND ts < :ts ORDER BY ts DESC LIMIT :count", [":ts" := ts, ":count" := count])
-
-type ChatStatsRow = (Int, ChatItemId, Bool)
+type ChatStatsRow = (Int, Int, ChatItemId, BoolInt)
 
+-- TODO [reports]
```

Comment:
```suggestion
```

Reply:
or is it not done? comments in Messages.hs imply it's read / not read by design at this point